media/js/firefox/new/scene2.js

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// set up namespace
+if (typeof Mozilla == 'undefined') {
+    var Mozilla = {};
+}
+
+Mozilla.Firefox = window.Mozilla.Firefox || {};
+
+;(function($, dataLayer, MFNew) {
+    'use strict';
+
+    var $directDownloadLink = $('#direct-download-link');
+    var $stage = $('#stage');
+    var downloadStarted = parseInt(MFNew.params.get('dlstarted'), 10) === 1;
+
+    // pixel remains in perpetuity (https://bugzilla.mozilla.org/show_bug.cgi?id=1222945#c2)
+    if (!window._dntEnabled()){
+        var $body = $('body');
+
+        var $pixel = $('<img />', {
+            width: '1',
+            height: '1',
+            src: 'https://servedby.flashtalking.com/spot/8/6247;40428;4669/?spotName=Mozilla_Download_Conversion'
+        });
+
+        $body.append($pixel);
+    }
+
+    // Pull download link from the download button and add to the
+    // 'click here' link.
+    // TODO: Remove and generate link in bedrock.
+    $directDownloadLink.attr(
+        'href', $('#download-button-wrapper-desktop .download-list li:visible .download-link').attr('href')
+    );
+
+    // #direct-download-link = "click here" text on page
+    // .download-link = any links in download button (which are effectively
+    // hidden, but could be clicked by screen reader?)
+    $stage.on('click', '#direct-download-link, .download-link', function(e) {
+        e.preventDefault();
+
+        var url = $(e.currentTarget).attr('href');
+
+        // An iframe cannot be used here to trigger the download because
+        // it will be blocked by Chrome if the download link redirects
+        // to a HTTP URI and we are on HTTPS.
+        function trackAndRedirect(url, virtualUrl) {
```

Comment:
Does keeping this as a function still serve a purpose now that it's only called once within this block?